Dev vulcan (#562)
taptap/tds-docs: i18n/en/docusaurus-plugin-content-docs/current/sdk/start/faq.mdx

For exclusive games, TDS provides high protection free of charge during the exclusive period. Developers simply need to communicate with TDS in advance about configuration details.
For other games, TDS can assist in accessing the IaaS provider's DDoS protection service at the developer's expense (IaaS providers charge for DDoS protection bandwidth). Please contact us for further details.

> ### Is there a limit to the number of accounts with real-name authentication for the same ID card?

Reviewer: ```suggestion
### Is there a limit to the number of accounts with real-name authentication for the same ID?
```

> Currently, TapTap requires that a single ID card can only be used for real-name authentication on up to 5 TapTap accounts. There is no limit on real-name authentication for game accounts.

Reviewer: ```suggestion
Currently, TapTap requires that a single ID can only be used for real-name authentication on up to 5 TapTap accounts. There is no limit on real-name authentication for game accounts.
```

```
The error is caused by using TapSDK Unity v3.7.1 and above without adding the `com.leancloud.storage` module to the project's `Packages/manifest.json` file. See [documentation](/sdk/taptap-login/guide/tap-login/#getting-the-sdk).

> ### Login prompt: this app is not allowed for this domain

Reviewer: ```suggestion
### The player sees this message when trying to log in: this app is not allowed for this domain
```

> - 1. Check whether the Tap login service is enabled in the application configuration in the developer console;
> - 2. Ensure that the Client id, ClientToken, and ServerUrl (must start with https://) in the project initialization code are consistent with those in the developer console.

Reviewer: ```suggestion
1. Check whether the TapTap login service is enabled in the application configuration in the developer console;
2. Ensure that the ClientId, ClientToken, and ServerUrl (must start with `https://`) in the project initialization code are consistent with those in the developer console.
```

![](https://dc-file.leanticket.cn/VIFonJ9YOJ4SAXb3WdVhMYWlF84xGKkN/CF18E02C-C819-4940-9C9B-60050062EDD0.png)

## Anti-addiction

> ### Do individual/group developers need to implement anti-addiction measures?

Reviewer: ```suggestion
### Is it required for developers (including organizations) to implement anti-addiction measures?
```

> According to the Notice of Further Strict Management and Effective Prevention of Minors' Addiction to Online Games issued by China's National Press and Publication Administration [link](https://www.nppa.gov.cn/xxfb/tzgs/202108/t20210830_666285.html), all game publishing and operating companies must implement real-name authentication and anti-addiction policies within their games. Games without a license can use TDS's [Real-name Authentication Anti-addiction](/sdk/anti-addiction/features/) system.

Reviewer: ```suggestion
According to the Notice of Further Strict Management and Effective Prevention of Minors' Addiction to Online Games issued by China's National Press and Publication Administration [link](https://www.nppa.gov.cn/xxfb/tzgs/202108/t20210830_666285.html), all game publishing and operating companies must implement real-name authentication and anti-addiction policies within their games. Games without a license can use TDS's [Real-name Authentication and Anti-addiction](/sdk/anti-addiction/features/) system.
```

### What is the relationship between anti-addiction and login?

> Starting from version 3.7.1, quick authentication does not rely on TapTap Login SDK, so there is **no** strong association between anti-addiction and login. However, after using TAPTAP login, you can use [Quick Authentication](/sdk/anti-addiction/features/#taptap-quick-verification) for anti-addiction. With the player's consent, they can quickly complete the in-game authentication process using their real-name information verified by TapTap.

Reviewer: ```suggestion
Starting from version 3.7.1, Quick Verification does not rely on the TapTap Login SDK, so there is **no** strong association between anti-addiction and login. However, after using TapTap login, you can use [Quick Verification](/sdk/anti-addiction/features/#taptap-quick-verification) for anti-addiction. With the player's consent, they can quickly complete the in-game authentication process using their real-name information verified by TapTap.
```

> **Otherwise**, you can use our provided [Real-name Authentication](/sdk/anti-addiction/features/#real-name-verification-and-anti-addiction). TDS has automatically connected to the Ministry of Publicity's online game anti-addiction real-name authentication system and reports information under the developer's name, meeting regulatory requirements.

Reviewer: ```suggestion
**Otherwise**, you can use our provided [Real-name Authentication](/sdk/anti-addiction/features/#real-name-verification-and-anti-addiction). TDS has connected to the Ministry of Publicity's online game anti-addiction and real-name authentication system and reports information under the developer's name, meeting regulatory requirements.
```

> ### After exiting an account due to anti-addiction, the game is logged in directly when restarted.

Reviewer: ```suggestion
### After logging out an account that has completed the real-name verification and restarting the game, the real-name verification process is skipped.
```

> Starting from version 3.7.1, quick authentication does not rely on TapTap Login SDK, so there is **no** strong association between anti-addiction and login. Therefore, you need to log out of the current account before switching accounts. The first time authentication will trigger a real-name authentication popup for players to authorize the game to access their TapTap real-name information or input their identity information. After successful authentication, subsequent calls to the authentication interface with the same userIdentifier (unique identifier) will directly receive the result of the first-time authentication without triggering a popup.

Reviewer: ```suggestion
Starting from version 3.7.1, quick authentication does not rely on TapTap Login SDK, so there is **no** strong association between anti-addiction and login. Therefore, you need to log out of the current account before switching accounts. When a player logs in for the first time, the SDK will show a real-name verification popup for players to authorize the game to access their TapTap real-name information or input their identity information. After successful authentication, subsequent calls to the authentication interface with the same userIdentifier (unique identifier) will directly receive the result of the first-time authentication without triggering a popup.
```

**Otherwise**, you can use our provided [Real-name Authentication](/sdk/anti-addiction/features/#real-name-verification-and-anti-addiction). TDS has automatically connected to the Ministry of Publicity's online game anti-addiction real-name authentication system and reports information under the developer's name, meeting regulatory requirements.

### After exiting an account due to anti-addiction, the game is logged in directly when restarted.
Starting from version 3.7.1, quick authentication does not rely on TapTap Login SDK, so there is **no** strong association between anti-addiction and login. Therefore, you need to log out of the current account before switching accounts. The first time authentication will trigger a real-name authentication popup for players to authorize the game to access their TapTap real-name information or input their identity information. After successful authentication, subsequent calls to the authentication interface with the same userIdentifier (unique identifier) will directly receive the result of the first-time authentication without triggering a popup.

> Thus, a user's unique identifier should be ensured for uniqueness.

Reviewer: 这里似乎多删了一行？
